types/react/experimental.d.ts

```diff
@@ -164,9 +164,6 @@ declare module '.' {
      */
     export function unstable_useTransition(config?: SuspenseConfig | null): [TransitionStartFunction, boolean];
```

Comment:
Do we need to change types here now that `timeoutMs` is removed?

see https://github.com/facebook/react/pull/19703

Reply:
Not including updates for experimental release channel. Goal is to have a minimal update that can be shipped as soon as possible.

Reply:
@maraisr I suspect this PR takes longer since a maintainer has to manually unblock it (benchmark failing due to timeout) so I opened https://github.com/DefinitelyTyped/DefinitelyTyped/pull/49152 for `@experimental`.

Reply:
Released in `@types/react@16.9.54`